### PR TITLE
[NIC-25] Add get project and service by id

### PR DESCRIPTION
**Pull request by AksyonovaKaterina**

### CODE REVIEW

File: backend/person-service/src/main/kotlin/com/nosql/personservice/controller/OfficeController.kt [hunk 1]
```diff
@@ -19,7 +20,17 @@ class OfficeController(
...
-        produces = [MediaType.APPLICATION_JSON_VALUE],
+        produces = [APPLICATION_JSON_VALUE],
     )
     suspend fun getAll(@PageableDefault(page = 0, size = 20) pageable: Pageable) = officeService.getAll(pageable)
+
+    @GetMapping(
+        value = [OFFICE_ID_URL_PATH],
+        produces = [APPLICATION_JSON_VALUE],
+    )
+    suspend fun get(@PathVariable id: String) = officeService.get(id)
```
Comment (V7074) [inline backend/person-service/src/main/kotlin/com/nosql/personservice/controller/OfficeController.kt]:
```suggestion
    suspend fun getById(@PathVariable id: String) = officeService.get(id)
```

File: backend/person-service/src/main/kotlin/com/nosql/personservice/controller/ProjectController.kt [hunk 2]
```diff
@@ -22,4 +23,14 @@ class ProjectController(
         produces = [MediaType.APPLICATION_JSON_VALUE],
     )
     suspend fun getAll(@PageableDefault(page = 0, size = 20) pageable: Pageable) = projectService.getAll(pageable)
+
+    @GetMapping(
+        value = [PROJECT_ID_URL_PATH],
+        produces = [MediaType.APPLICATION_JSON_VALUE],
```
Comment (V7074) [inline backend/person-service/src/main/kotlin/com/nosql/personservice/controller/ProjectController.kt]: static import

File: backend/person-service/src/main/kotlin/com/nosql/personservice/controller/ProjectController.kt [hunk 3]
```diff
@@ -22,4 +23,14 @@ class ProjectController(
...
+    )
+    suspend fun get(@PathVariable id: String) = projectService.get(id)
```
Comment (V7074) [inline backend/person-service/src/main/kotlin/com/nosql/personservice/controller/ProjectController.kt]:
```suggestion
    suspend fun getById(@PathVariable id: String) = projectService.get(id)
```

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/OfficeService.kt [hunk 4]
```diff
@@ -5,5 +5,7 @@ import org.springframework.data.domain.Pageable
 
 interface OfficeService {
 
+    suspend fun get(id: String): OfficeDto
```
Comment (V7074) [inline backend/person-service/src/main/kotlin/com/nosql/personservice/service/OfficeService.kt]: same

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/ProjectService.kt [hunk 5]
```diff
@@ -5,5 +5,7 @@ import org.springframework.data.domain.Pageable
 
 interface ProjectService {
 
+    suspend fun get(id: String): ProjectDto
```
Comment (V7074) [inline backend/person-service/src/main/kotlin/com/nosql/personservice/service/ProjectService.kt]: same

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultOfficeService.kt [hunk 6]
```diff
@@ -14,6 +15,9 @@ class DefaultOfficeService(
     private val conversionService: ConversionService,
 ) : OfficeService {
 
+    override suspend fun get(id: String) = officeComponent.get(ObjectId(id))
```
Comment (V7074) [inline backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultOfficeService.kt]: same

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultProjectService.kt [hunk 7]
```diff
@@ -14,6 +15,9 @@ class DefaultProjectService(
     private val conversionService: ConversionService,
 ) : ProjectService {
 
+    override suspend fun get(id: String) = projectComponent.get(ObjectId(id))
```
Comment (V7074) [inline backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultProjectService.kt]: same

and check the other places, please

**Review by V7074 — CHANGES_REQUESTED** [review]

File: backend/person-service/src/main/kotlin/com/nosql/personservice/controller/OfficeController.kt (hunk 1 above)
Comment (AksyonovaKaterina) [reply backend/person-service/src/main/kotlin/com/nosql/personservice/controller/OfficeController.kt]: fixed

File: backend/person-service/src/main/kotlin/com/nosql/personservice/controller/ProjectController.kt (hunk 2 above)
Comment (AksyonovaKaterina) [reply backend/person-service/src/main/kotlin/com/nosql/personservice/controller/ProjectController.kt]: fixed

File: backend/person-service/src/main/kotlin/com/nosql/personservice/controller/ProjectController.kt (hunk 3 above)
Comment (AksyonovaKaterina) [reply backend/person-service/src/main/kotlin/com/nosql/personservice/controller/ProjectController.kt]: fixed

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/ProjectService.kt (hunk 5 above)
Comment (AksyonovaKaterina) [reply backend/person-service/src/main/kotlin/com/nosql/personservice/service/ProjectService.kt]: fixed

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultOfficeService.kt (hunk 6 above)
Comment (AksyonovaKaterina) [reply backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultOfficeService.kt]: fixed

File: backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultProjectService.kt (hunk 7 above)
Comment (AksyonovaKaterina) [reply backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultProjectService.kt]: fixed

**Review by V7074 — APPROVED** [review]

**Pull request merged**